docs/notebooks/utils/view_swc_spline.ipynb

```diff
@@ -0,0 +1,203 @@
+{
```

Comment:
I will put comments for the notebook here:

- brainlit.algorithms.connect_fragments.fit_spline doesn't exist anymore
- Change "fit_spline_tree..." title to "Fitting Splines to Neuron Trace SWC Tutorial"
- Have the notebook read [this file](https://github.com/NeuroDataDesign/brainlit/tree/develop/data/data_octree/consensus-swcs) rather than "C:/Users/..."
- "extrated" typo
- After [this issue](https://github.com/neurodata/brainlit/issues/164) is addressed, please replace the code below 2) with a one-liner
- Make the neuron figure in 3) prettier (the red ball at the soma is obnoxious) and have a markdown cell explaining what's being shown 
- In 4), only plot a few branches, not every single one

Reply:
dont worry about bullet 5

Reply:
1. path to ```fit_spline```modified
2. title changed
3. **not finished**
4. pyto fixed
5. skip
6. plots modified
7. reduce to 3 splines

Reply:
3. Added a [function](https://github.com/NeuroDataDesign/brainlit/blob/CTSprint1/docs/notebooks/algorithms/view_swc_spline.ipynb?short_path=927559d#L60-L81) to search for the swc file locally. It can fail when the current directory is lower than ```brainlit```folder.